packages/mobile/src/fees/saga.ts

```diff
@@ -23,88 +37,183 @@ export interface FeeInfo {
   currency: Currency
 }
 
-// TODO(victor): This fee caching mechansim is only being used by the balance check on the send
-// amount entry screen. In an effort to standardize and improve fee estimation, we should either
-// update and use this mechanism everywhere or remove it in favor of another solution.
-// Cache of the gas estimates for common tx types
-// Prevents us from having to recreate txs and estimate their gas each time
-const feeGasCache = new Map<FeeType, BigNumber>()
```

Comment:
Are we still caching fees?

Reply:
Yes, but in redux. This was I think legacy or something, it was storing fees in this map which is not a pattern we use anywhere else. The comment said `In an effort to standardize and improve fee estimation, we should either
update and use this mechanism everywhere or remove it in favor of another solution.` I chose the second option, removing it in favor of storing fee estimations on Redux